doc/whatsnew/2.14.rst

```diff
@@ -278,6 +278,8 @@ Other Changes
 
   Closes #5608
 
+* Updated the default values of ``ignore-imports`` and ``ignore-signatures`` to be ``True``.
+
```

Comment:
```suggestion
```suggestion
* By default the similarity checker will now ignore imports and ignore function signatures when computing
 duplication. If you want to keep the previous behaviour set ``ignore-imports`` and ``ignore-signatures`` to ``False``.
```

```